runtime/src/accounts.rs

```diff
@@ -167,22 +167,27 @@ pub struct Accounts {
 
     /// List of persistent stores
     paths: String,
+
+    /// paths was not specified, so defaults were used
+    made_default_paths: bool,
```

Comment:
I don't understand this variable name or the comment that describes it. Can you clarify?

Reply:
When you create the Accounts object you can pass in paths to use or None and it generates the default paths like `target/<random-string>/0/`. This variable is set to true in the latter case, and we use it to clean up the directories later, since we created multiple levels, it will be safe to clean it up.

What do you suggest for naming?

Reply:
`own_paths`?  How about a description like, "Set to true if this object created the directories in 'paths'. When true, delete 'paths' when this object is dropped."